apps/stirling-pdf/0.35.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.35.0
+    image: frooodle/s-pdf:0.35.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is no change between your code snippet and the provided example in terms of differences or key components compared against a specific knowledge cutoff from September 1st, 2021.

If you have any other questions about Kubernetes services configuration, deployments etc., feel free to ask!